src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.String.cs

```diff
@@ -41,6 +46,36 @@ public static partial class JsonSerializer
                 throw new ArgumentNullException(nameof(json));
             }
 
+            return Deserialize<TValue>(json.AsSpan(), typeof(TValue), options);
+        }
+
+        /// <summary>
+        /// Parse the text representing a single JSON value into an instance of a specified type.
```

Comment:
This method is generic, there is no type parameter being specified.
Borrowing the wording from the existing string overload:
https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializer.deserialize?view=netcore-3.1#System_Text_Json_JsonSerializer_Deserialize__1_System_String_System_Text_Json_JsonSerializerOptions_

https://github.com/dotnet/dotnet-api-docs/blob/fcf78e35bc4eca5734d3f248670b1ade805e2ed0/xml/System.Text.Json/JsonSerializer.xml#L296

```suggestion
        /// Parses the text representing a single JSON value into an instance of the type specified by a generic type parameter.
```